packages/e2e-tests/specs/editor/various/embedding.test.js

```diff
@@ -278,6 +278,6 @@ describe( 'Embedding content', () => {
 		await insertEmbed( postUrl );
 
 		// Check the block has become a WordPress block.
-		await page.waitForSelector( '.wp-block-embed-wordpress' );
+		await page.waitForSelector( 'figure.wp-block-embed' );
```

Comment:
// Is there a way to better test some of these tests? Maybe with `is-provider-...` or some other class? I tried //`saveDraft` and `publishPost` and get the element again, but these classes are not there.
// the `is-provider-...` class is added in `save.js`

Reply:
Maybe combine these `waitForSelector` calls with a general `expect` at the end of the test that matches `getEditedPostContent` against a snapshot? As a benefit, this also targets some other pieces that were left unchecked like:

```js
// Photo content. Should render valid figure element.
await insertEmbed( 'https://cloudup.com/cQFlxqtY4ob' );
```

Reply:
Now looking at these files again, I notice the lack of `wp-block-embed-${type}` classes, as I pointed out in the other comment. Sorry I didn't point this out earlier.